From 33f04ca8a23c0fcc2d4260958c3185f4fc3d3d66 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 27 Jun 2023 10:28:44 +0200
Subject: [PATCH] man: rebreak lines in file-hierarchy(7) a bit

(Does not change a single word, just rebreaks a bunch of paragraphs
matching our current line breaking rules)

(cherry picked from commit fa1d34825a9b410275e716b9b70f4fca02c71ba9)

Related: RHEL-137252
---
 man/file-hierarchy.xml | 47 ++++++++++++++++--------------------------
 1 file changed, 18 insertions(+), 29 deletions(-)

diff --git a/man/file-hierarchy.xml b/man/file-hierarchy.xml
index 6943f386bc..6104324519 100644
--- a/man/file-hierarchy.xml
+++ b/man/file-hierarchy.xml
@@ -505,13 +505,10 @@
       <varlistentry>
         <term><filename>~/.cache/</filename></term>
 
-        <listitem><para>Persistent user cache data. User programs may
-        place non-essential data in this directory. Flushing this
-        directory should have no effect on operation of programs,
-        except for increased runtimes necessary to rebuild these
-        caches. If an application finds
-        <varname>$XDG_CACHE_HOME</varname> set, it should use the
-        directory specified in it instead of this
+        <listitem><para>Persistent user cache data. User programs may place non-essential data in this
+        directory. Flushing this directory should have no effect on operation of programs, except for
+        increased runtimes necessary to rebuild these caches. If an application finds
+        <varname>$XDG_CACHE_HOME</varname> set, it should use the directory specified in it instead of this
         directory.</para></listitem>
       </varlistentry>
 
@@ -527,44 +524,36 @@
       <varlistentry>
         <term><filename>~/.local/bin/</filename></term>
 
-        <listitem><para>Executables that shall appear in the user's
-        <varname>$PATH</varname> search path. It is recommended not to
-        place executables in this directory that are not useful for
-        invocation from a shell; these should be placed in a
-        subdirectory of <filename>~/.local/lib/</filename> instead.
-        Care should be taken when placing architecture-dependent
-        binaries in this place, which might be problematic if the home
-        directory is shared between multiple hosts with different
+        <listitem><para>Executables that shall appear in the user's <varname>$PATH</varname> search path. It
+        is recommended not to place executables in this directory that are not useful for invocation from a
+        shell; these should be placed in a subdirectory of <filename>~/.local/lib/</filename> instead. Care
+        should be taken when placing architecture-dependent binaries in this place, which might be
+        problematic if the home directory is shared between multiple hosts with different
         architectures.</para></listitem>
       </varlistentry>
 
       <varlistentry>
         <term><filename>~/.local/lib/</filename></term>
 
-        <listitem><para>Static, private vendor data that is compatible
-        with all architectures.</para></listitem>
+        <listitem><para>Static, private vendor data that is compatible with all
+        architectures.</para></listitem>
       </varlistentry>
 
       <varlistentry>
         <term><filename>~/.local/lib/<replaceable>arch-id</replaceable>/</filename></term>
 
-        <listitem><para>Location for placing public dynamic libraries.
-        The architecture identifier to use is defined on <ulink
-        url="https://wiki.debian.org/Multiarch/Tuples">Multiarch
-        Architecture Specifiers (Tuples)</ulink>
-        list.</para></listitem>
+        <listitem><para>Location for placing public dynamic libraries. The architecture identifier to use is
+        defined on <ulink url="https://wiki.debian.org/Multiarch/Tuples">Multiarch Architecture Specifiers
+        (Tuples)</ulink> list.</para></listitem>
       </varlistentry>
 
       <varlistentry>
         <term><filename>~/.local/share/</filename></term>
 
-        <listitem><para>Resources shared between multiple packages,
-        such as fonts or artwork. Usually, the precise location and
-        format of files stored below this directory is subject to
-        specifications that ensure interoperability. If an application
-        finds <varname>$XDG_DATA_HOME</varname> set, it should use the
-        directory specified in it instead of this
-        directory.</para></listitem>
+        <listitem><para>Resources shared between multiple packages, such as fonts or artwork. Usually, the
+        precise location and format of files stored below this directory is subject to specifications that
+        ensure interoperability. If an application finds <varname>$XDG_DATA_HOME</varname> set, it should use
+        the directory specified in it instead of this directory.</para></listitem>
       </varlistentry>
 
       <varlistentry>
